src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -186,5 +190,21 @@ define(function (require, exports, module) {
             });
 
         });
+	}
+
+
+    function appReady() {
+        addEnabledPref()    
```

Comment:
At this point, because it's a single line function, I think you should just inline this and put `preferences.definePreference("enabled", "boolean", true);` here vs. calling the function.